diff --git a/clearpath_platform_description/urdf/generic/gazebo.urdf.xacro b/clearpath_platform_description/urdf/generic/gazebo.urdf.xacro
index 612b883..3c29602 100644
--- a/clearpath_platform_description/urdf/generic/gazebo.urdf.xacro
+++ b/clearpath_platform_description/urdf/generic/gazebo.urdf.xacro
@@ -5,15 +5,15 @@
       <plugin filename="libign_ros2_control-system.so" name="ign_ros2_control::IgnitionROS2ControlPlugin">
         <parameters>$(arg gazebo_controllers)</parameters>
         <ros>
-          <remapping>~/odom:=platform/odom</remapping>
-          <remapping>~/cmd_vel_unstamped:=platform/cmd_vel_unstamped</remapping>
+          <remapping>~/odom:=odom</remapping>
+          <remapping>~/cmd_vel_unstamped:=cmd_vel</remapping>
           <remapping>/tf:=tf</remapping>
           <remapping>/tf_static:=tf_static</remapping>
           <remapping>/diagnostics:=diagnostics</remapping>
-          <remapping>/dynamic_joint_states:=platform/dynamic_joint_states</remapping>
-          <remapping>joint_states:=platform/joint_states</remapping>
-          <remapping>~/reference:=platform/cmd_vel</remapping>
-          <remapping>~/odometry:=platform/odom</remapping>
+          <remapping>/dynamic_joint_states:=dynamic_joint_states</remapping>
+          <remapping>joint_states:=joint_states</remapping>
+          <remapping>~/reference:=cmd_vel</remapping>
+          <remapping>~/odometry:=odom</remapping>
           <namespace>$(arg namespace)</namespace>
         </ros>
       </plugin>
